util.js

```diff
@@ -0,0 +1,65 @@
+var Command = require('./command');
+var debug = require('debug')('mozdevice:util');
+
+/**
+ * Miscella
+ * @param device
+ * @constructor
+ */
+var Util = function(device) {
+  this.serial = device.serial;
+};
+
+/**
+ * Reboot the device and fetch the device time at the moment it occurred
+ * @returns {Promise}
+ */
+Util.prototype.reboot = function() {
+  debug('Rebooting');
+
+  return new Command()
+    .env('DEVICETIME',
+      "$(ANDROID_SERIAL=" + this.serial + " adb shell echo '$EPOCHREALTIME')")
+    .and()
+      .env('ANDROID_SERIAL', this.serial)
+      .adb('reboot')
+    .and()
+      .env('ANDROID_SERIAL', this.serial)
+      .adb('wait-for-device')
+    .and()
+      .echo('$DEVICETIME')
+    .exec()
+    .then(function(stdout) {
+      // The time generated by EPOCHREALTIME is seconds, we need milliseconds
+      return parseFloat(stdout.replace('\n', '')) * 1000;
```

Comment:
Do you not want to mark the reboot? Previously (in raptor/lib/device) there was:
Device.mark('deviceReboot@System', time);